src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -0,0 +1,348 @@
+   * work on the element after update.
+   *
+   * Setting properties inside this method will trigger the element to update
+   * again after this update cycle completes.
+   *
+   * * @param _changedProperties Map of changed properties with old values
+   */
+  public firstUpdated() {
+    Providers.onProviderUpdated(() => this.loadData());
+    this.loadData();
+  }
+
+  /**
+   * Invoked on each update to perform rendering tasks. This method must return
+   * a lit-html TemplateResult. Setting properties inside this method will *not*
+   * trigger the element to update.
+   */
+  protected render() {
+    if (this.personDetails) {
+      const user = this.personDetails;
+
+      // tslint:disable-next-line: one-variable-per-declaration
+      let department, jobTitle;
+
+      if (user.department) {
+        department = html`
+          <div class="department">${user.department}</div>
+        `;
+      }
+
+      if (user.jobTitle) {
+        jobTitle = html`
+          <div class="job-title">${user.jobTitle}</div>
+        `;
+      }
+      return html`
+        <div class="root" @click=${this.handleClose}>
+          <div class="default-view">
+            ${this.renderTemplate('default', { person: this.personDetails }) ||
+              html`
+                <mgt-person .personDetails=${this.personDetails} .personImage=${this.personImage}></mgt-person>
+                <div class="details">
+                  <div class="display-name">${user.displayName}</div>
+                  ${jobTitle} ${department}
+                  <div class="base-icons">
+                    ${this.renderIcons()}
+                  </div>
+                </div>
+              `}
+          </div>
+          <div class="additional-details-container" @click=${this._showAdditionalDetails}>
+            ${this.renderAdditionalDetails()}
+          </div>
+        </div>
+      `;
+    }
+  }
+
+  private renderIcons() {
+    const user = this.personDetails;
+    // tslint:disable-next-line: one-variable-per-declaration
+    let chat, email, phone;
+    if ((user as MicrosoftGraph.User).mailNickname) {
+      chat = html`
+        <div @mouseout=${this._unsetMouseOverState} @mouseover=${this._setMouseOverState} @click=${this._chatUser}>
+          ${svgHelper.getSVG('chat', '#666666')}
+        </div>
+      `;
+    }
+    if (getEmailFromGraphEntity(user)) {
+      email = html`
+        <div @mouseout=${this._unsetMouseOverState} @mouseover=${this._setMouseOverState} @click=${this._emailUser}>
+          ${svgHelper.getSVG('email', '#666666')}
+        </div>
+      `;
+    }
+
+    if ((user as MicrosoftGraph.User).businessPhones && (user as MicrosoftGraph.User).businessPhones.length > 0) {
+      phone = html`
+        <div @mouseout=${this._unsetMouseOverState} @mouseover=${this._setMouseOverState} @click=${this._callUser}>
+          ${svgHelper.getSVG('phone', '#666666')}
+        </div>
+      `;
+    }
+
+    if (this.isExtended === true) {
```

Comment:
just a minor thing, but more readable. You don't need to set the values of the chat, email, and phone vars if it's extended. Check if extended at the beginning of the function and just return if it is.

```
private renderIcons() {
    if (this.isExtended === true) {
        return html``;
    }

    const user = this.personDetails;
    // rest of function
```

Reply:
refactored: 

273ae5a